### ISSUE TITLE
Error when Cache-Control is not set

### ISSUE THREAD

**Issue by hjr3** [falsandtru/pjax-api#40]
I get the following error when `Cache-Control` header is not set:

```
Uncaught TypeError: Cannot read property 'trim' of null
    at pjax-api.js:3905
    at pjax-api.js:655
    at Either.thunk (pjax-api.js:667)
    at Either.evaluate (pjax-api.js:800)
    at Either.thunk (pjax-api.js:662)
    at Either.evaluate (pjax-api.js:800)
    at Either.thunk (pjax-api.js:662)
    at Either.evaluate (pjax-api.js:800)
    at Either.extract (pjax-api.js:679)
    at Either.extract (pjax-api.js:679)
```

Example response:

```
HTTP/1.1 200 OK
ETag: W/"1500-bBhQrFKrKDiHMxhJ8mHxWb5w21U"
Date: Fri, 17 May 2019 21:38:24 GMT
Connection: keep-alive
```

Express is setting the `ETag` header but does not automatically set the `Cache-Control` header. The work around is to set a `Cache-Control` header.

I believe the issue is that Cache-Control is not being defaulted to `''` on this [line](https://github.com/falsandtru/pjax-api/blob/v3.29.5/src/layer/domain/router/module/fetch/xhr.ts#L63), but is defaulted to `''` just above in the _if_ check.

**Comment by falsandtru** [comment]
Thanks, I published the current fixed code to npm as v3.29.6.

**Comment by hjr3** [comment]
Thank you for the quick response and great library!
